src/Compute/Compute/ChangeLog.md

```diff
@@ -28,6 +28,7 @@
 * Added `Hibernate` parameter to `Stop-AzVm` cmdlet. 
 * Added `HibernationEnabled` parameter to `New-AzVm`, `New-AzVmConfig`, and `Update-AzVm` cmdlets.
 * Added `EnableHotpatching` parameter to the `Set-AzVmssOSProfile` cmdlet.
+* Updated `New-AzureSshKey` cmdlet to write file paths to generated keys to the Warning stream instead of the console.
```

Comment:
Please move this to `Upcoming Release` section. We update the version these days, some context is changed.

Reply:
Moved to Upcoming Release, I completely missed that. Thank you!